data/levels.yml

```diff
@@ -1,4 +1,14 @@
 
+# Junior Engineer
+- id: junior
+  name: Junior Engineer
+  summary: >
+    Junior Engineers are expected to work mostly independently on areas of
```

Comment:
Excellent. One suggestion is rather than 'seek to learn', 'demonstrate continuous learning' (or something less wordy with a similar outcome rather than intent skew)

Reply:
Good point. I'll change it to "learn new skills", which is more active.